subprojects/android/android.gradle

```diff
@@ -0,0 +1,57 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+description = "Mockito for Android"
+
+Properties props = new Properties()
+props.load(new FileInputStream(rootProject.file("version.properties")))
+
+group = 'org.mockito'
+archivesBaseName = 'mockito-android'
+version = props['mockito.version']
+
+def licenseFiles = copySpec {
+    //mockito license
+    from(".") { include 'LICENSE' }
+}
+
+jar {
+    baseName = archivesBaseName
+    from(sourceSets.main.allSource)
+    with licenseFiles
+}
+
+task sourcesJar(type: Jar, dependsOn: classes) {
+    classifier = "sources"
+    from sourceSets.main.allJava.srcDirs
+    include "**/*.java", "**/*.aj"
+}
+
+artifacts {
+    archives sourcesJar
+}
+
+tasks.javadoc.enabled = true
+
+publishing {
+    publications {
+        mockitoAndroid(MavenPublication) {
+            from components.java
+            artifactId archivesBaseName
+            artifact sourcesJar
+            artifact javadocJar
+        }
+    }
+}
```

Comment:
The `publishing` declaration here fail, not sure why https://scans.gradle.com/s/7myhdoeojp3wi/failure
as it does look correct.